base/steer/FairRunSim.h

```diff
@@ -232,6 +242,7 @@ class FairRunSim : public FairRun
     TString fUserConfig;                            //!                        /** Macro for geant configuration*/
     TString fUserCuts;                              //!                          /** Macro for geant cuts*/
     Bool_t fIsMT;                                   //!                              /** MT mode option (Geant4 only)*/
+    Bool_t fWasMT;                                  //!                              /** Actual MT mode used */
```

Comment:
Maybe?

```suggestion
    bool fWasMT{false};                             //!                              /** Actual MT mode used */
```

And I think, you might even move that to the `private:` section?

Reply:
Please initialize `fWasMT` (probably to `false`), so that `WasMT()` returns a defined value even before a run?